docker-compose.yaml

```diff
@@ -39,8 +39,8 @@ services:
 
   rethink:
     image: rethinkdb:2.3
-    expose:
-      - 28015
+    ports:
+      - "9090:8080"
```

Comment:
Hi @grkvlt Thanks for your work on this. Does the RethinkDb container need to link ports with the host and are these the correct ports?

Reply:
This is for the RethinkDB dashboard, so I exposed it to have a look at how it works. I think it's fine for the compose files, obviously you'd do something a bit more secure for a production setup...